integration/helper_test.go

```diff
@@ -433,6 +434,116 @@ options:
 			Expect(err).To(HaveOccurred())
 			assertErrorsHandled()
 		})
+
+		It("skips batches if skip file discovered", func() {
+			// Run helper only with restore for a few batches and skip file defined
+			//
+			By("Write data file")
+			dataFile := dataFileFullPath
+			f, _ := os.Create(dataFile + ".gz")
```

Comment:
why not just
```
			f, _ := os.Create(dataFileFullPath + ".gz")
```
? Or maybe
```
			dataFile := dataFileFullPath + ".gz"
			f, _ := os.Create(dataFile)
```
? Then we can reuse `dataFile` instead of `dataFileFullPath+".gz"` below in the test...